debian/copyright

```diff
@@ -0,0 +1,38 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: coding-shell-extensions
+Source: <url://example.com>
+
+Files: *
+Copyright: <years> <put author's name and email here>
```

Comment:
so seems like the `copyright` file itself is required https://www.debian.org/doc/manuals/maint-guide/dreq.en.html#copyright but only the `format` field in the `copyright` file is required https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#format-field

So will get rid of everything else. 👍